### DISCUSSION TITLE
shortest path

### DISCUSSION THREAD

**Discussion by Ranjith2002** [JanusGraph/janusgraph#3657] [General]
sir we can find nth shortest path in the graph. like if a to b has 7 paths available. if we need shortest path means we can use shortestPath(). but we need second or third or nth shortest path means how to find . is it possible?

**Comment by hadoopmarc** [comment]
Sure, take a look at:
https://tinkerpop.apache.org/docs/current/recipes/#shortest-path  

This resource also presents an OLTP approach as an alternative to the OLAP approach of the shortestPath() step.
